### ISSUE TITLE
CLI az acr build failure

### ISSUE THREAD

**Issue by feldkma** [Azure/azure-cli#18784]
## Describe the bug

Basic command to build and upload an image to ACR fails with little indication as to why.

**Command Name**
`az acr build`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
'NoneType' object is not callable
Traceback (most recent call last):
  File "/opt/az/lib/python3.6/site-packages/knack/cli.py", line 231, in invoke
    cmd_result = self.invocation.execute(args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 657, in execute
    raise ex
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 720, in _run_jobs_serially
    results.append(self._run_job(expanded_arg, cmd_copy))
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 691, in _run_job
    result = cmd_copy(params)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/__init__.py", line 328, in __call__
    return self.handler(*args, **kwargs)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/core/commands/command_operation.py", line 121, in handler
    return op(**command_args)
  File "/opt/az/lib/python3.6/site-packages/azure/cli/command_modules/acr/build.py", line 109, in acr_build
    variant=platform_variant
TypeError: 'NoneType' object is not callable

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az acr build --registry {} -t {} -t {} {}`

## Expected Behavior

Build a docker image and upload to ACR

## Environment Summary
```
Linux-4.4.0-96-generic-x86_64-with-debian-jessie-sid
Python 3.6.10
Installer: DEB

azure-cli 2.26.0

```

**Comment by feldkma** [comment]
When I use version `2.25.0-1~trusty`, I experience no problems.

**Comment by yonzhan** [comment]
route to service team

**Comment by monojit18** [comment]
Confirm that this is broken and noticing this from today morning!
On MacOS with HomeBrew install, I have no option to fallback on earlier versions as the brew repo is showing only 2.26.0 as the stable release.
As a workaround, I have to fallback onto docker build/push!!

**Comment by northtyphoon** [comment]
The issue has been fixed since 2.26.1
